File: platforms/web/examples/core/texture_mip_chain/base_size.js
import { TextureFormat, TextureMipChain } from "fragmentcolor";

const pixels = new Uint8Array(16 * 16 * 4);
const chain = TextureMipChain.prepare(pixels, TextureFormat.Rgba8UnormSrgb, [16, 16]);
const sz = chain.baseSize();
const width = sz.width;
const height = sz.height;

File: platforms/swift/examples/core/texture_mip_chain/base_size.swift
import FragmentColor

let pixels = Array(repeating: 0, count: 16 * 16 * 4)
let chain = try TextureMipChain.prepare((
    pixels,
    TextureFormat.rgba8UnormSrgb,
    [16, 16],
))
let size = chain.baseSize()
let _ = size